### PR TITLE
Second attempt at adding the tazer drone to the game

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
SUMMARY: Features "Adds the tazer drone to the game"

#### Purpose of change

This is the second attempt at adding a melee flying drone. Fast but fragile.

#### Describe the solution

This is a tazer drone (based off the tazer drone from Receiver 1. The shock is not insta-kill though.

#### Testing

Load in game, see it works. Spawned a drone, got zapped, shot it into pieces.

### CODE REVIEW

File: data/json/monsters/drones.json [hunk 1]
```diff
@@ -185,5 +185,19 @@
       "PRIORITIZE_TARGETS",
       "HIT_AND_RUN"
     ]
+  },
+  {
+    "id": "mon_tazer_hack",
+    "copy-from": "base_drone",
+    "type": "MONSTER",
+    "name": { "str": "tazer hack" },
+    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front. While extrememly fast, it is very fragile.",
```
Comment (BrettDong) [inline data/json/monsters/drones.json:194]:
```suggestion
    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front.  While extrememly fast, it is very fragile.",
```

File: data/json/items/corpses/inactive_bots.json [hunk 2]
```diff
@@ -740,5 +740,31 @@
       "moves": 150,
       "skills": [ "electronics", "computer" ]
     }
+  },
+  {
+    "id": "bot_tazer_hack",
+    "type": "TOOL",
+    "name": { "str": "inactive tazer hack" },
+    "description": "This is an inactive tazer hack.  tazer hacks are fist-sized robots that fly through the air.  This one has a tazer and attacks by flying at its target and delivering an electric shock.  Use this item to reprogram and release the tazer hack.  Electronics and computer skill determines if the targeting matrix is reprogrammed successfully.",
```
Comment (actual-nh) [inline data/json/items/corpses/inactive_bots.json:748]:
```suggestion
    "description": "This is an inactive tazer hack.  Hacks are fist-sized robots that fly through the air.  This one has a tazer and attacks by flying at its target and delivering an electric shock.  Use this item to reprogram and release the tazer hack.  Electronics and computer skills determine if the targeting matrix is reprogrammed successfully.",
```

File: data/json/monsters/drones.json [hunk 3]
```diff
@@ -185,5 +185,19 @@
       "PRIORITIZE_TARGETS",
       "HIT_AND_RUN"
     ]
+  },
+  {
+    "id": "mon_tazer_hack",
+    "copy-from": "base_drone",
+    "type": "MONSTER",
+    "name": { "str": "tazer hack" },
+    "description": "An automated drone, this small quadcopter robot appears to have a tazer strapped to the front.  While extrememly fast, it is very fragile.",
```
Comment (Maleclypse) [inline data/json/monsters/drones.json:194]: So it's makeshift strapped, not mass produced or manufactured as a complete product?  If makeshift who is making them and why?  I could absolutely see them as a product used in labs to subdue experiment subjects without killing them.  But then they'd be 3d printed to hold a tazer or some kind of similar design.

File: data/json/items/generic.json [hunk 4]
```diff
@@ -3346,5 +3346,23 @@
...
+  },
+  {
+    "type": "GENERIC",
+    "id": "broken_tazer_hack",
+    "symbol": ",",
+    "color": "yellow",
+    "name": { "str": "broken tazer hack" },
+    "category": "other",
+    "description": "A broken tazer hack, with its propellers still and tazer inert.  Could be gutted for parts.",
+    "price": 1000,
```
Comment (Maleclypse) [inline data/json/items/generic.json:3358]:
```suggestion
    "price": "10 USD",
```

File: data/json/monsters/drones.json (hunk 3 above)
Comment (Xenomorph-III) [reply data/json/monsters/drones.json]: Hm, yeah. That's a point of contention. Maybe "attached"

**Pull request closed without merging**